lib/aruba/process_monitor.rb

```diff
@@ -0,0 +1,113 @@
+module Aruba
```

Comment:
In this iteration I migrated all methods around starting, stopping, querying information about processes here. In the next iteration we can make more methods private I think.